roles/sap_ha_install_pacemaker/tasks/cluster_setup.yml

```diff
@@ -2,19 +2,37 @@
 # The following tasks are only included on the primary node!
 
 - name: "SAP Pacemaker Setup - Setup Pacemaker cluster"
-  ansible.builtin.shell: |
+  ansible.builtin.command: |
     pcs cluster setup \
     {{ sap_ha_install_pacemaker_cluster_name }} {% for node in sap_ha_install_pacemaker_cluster_nodes %}{{ node.node_name }}{{ " " if not loop.last else "" }}{% endfor %} \
-    totem token=30000 --force
+    totem token=30000 \
+    --enable --start --wait=30
+  register: __sap_ha_install_pacemaker_cluster_create
+  changed_when: __sap_ha_install_pacemaker_cluster_create.rc == 0
 
-- name: "SAP Pacemaker Setup - Start Pacemaker cluster"
+- name: "SAP Pacemaker Setup - Get cluster node corosync status"
   ansible.builtin.shell: |
-    pcs cluster start --all
-    pcs cluster enable --all
+    set -e -o pipefail
+    pcs status nodes corosync | grep "Online"
+  args:
+    executable: /usr/bin/bash
+  register: __sap_ha_install_pacemaker_cluster_online
+  failed_when: false
+  changed_when: false
 
-- name: "SAP Pacemaker Setup - Set Expected Votes = {{ sap_ha_install_pacemaker_cluster_nodes | length }}"
-  ansible.builtin.shell: |
-    pcs quorum expected-votes {{ sap_ha_install_pacemaker_cluster_nodes | length }}
+- name: "SAP PAcemaker Setup - Verify that all nodes are online"
```

Comment:
Uppercase typo `PAc...`

Reply:
Fixed in https://github.com/sap-linuxlab/community.sap_install/commit/4070477fadbd5bd9bf6b1fb2158b82c60204cec6